### DISCUSSION TITLE
posts/2023/02/added_giscus_commenting_to_the_blog

### DISCUSSION THREAD

**Discussion by giscus** [LeipeLeon/blog#3] [Comments]
# posts/2023/02/added_giscus_commenting_to_the_blog

https://www.wendbaar.nl/posts/2023/02/added_giscus_commenting_to_the_blog

**Comment by LeipeLeon** [comment]
Also check out these new notations in GHM (Github Flavoured Markdown):

```
> **Note**
> This is a note

> **Warning**
> This is a warning
```

> **Note**
> This is a note

> **Warning**
> This is a warning
